### DISCUSSION TITLE
how to solve 429 client error

### DISCUSSION THREAD

**Discussion by sq6er4** [TheR1D/shell_gpt#245] [Q&A] [answered]
when i use shell-gpt.it told me "HTTPERROR"
![image](https://github.com/TheR1D/shell_gpt/assets/88889900/fc953f6f-1671-4f49-9085-ccb0adf3f0d0)

**Comment by will-wright-eng** [comment, marked as answer]
https://github.com/TheR1D/shell_gpt/issues/201

**Reply by sq6er4** [reply]
thanks

**Comment by unphased** [comment]
Note I hit this and I signed up with my credit card but my token was still not working -- i had to make a new token after signing up for paid API access and it works with that one.
